Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Medical/beakers.yml

```
@@ -0,0 +1,71 @@
+- type: entity
+  parent: Beaker
+  id: BeakerRad
+  name: Beaker arithrazine
+  components:
+  - type: SolutionRegeneration
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)